src/hooks/protections/receive_net_message.cpp

```diff
@@ -236,7 +236,19 @@ namespace big
 			switch (msgType)
 			{
 			case rage::eNetMessage::MsgScriptMigrateHost: return true;
-			case rage::eNetMessage::MsgRadioStationSyncRequest: return false;
+			case rage::eNetMessage::MsgRadioStationSyncRequest:
+			{
+				if (unk_player_radio_requests.process())
+				{
+					if (unk_player_radio_requests.exceeded_last_process())
```

Comment:
Since this is for "all" people in the car, we should probably down the limit to 1-2 seconds. We need to test what happens if someone just presses next channel on the radio a ton of times.

Reply:
Ok, I have changed to 2s for now to be on the high side.